Example/CardStackExample/AppDelegate.m

```diff
@@ -67,7 +67,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 - (void)cardStackControllerWillOpen:(CardStackController *)cardStackController {
     CardView *card = [self.cardStackController.cards lastObject];
-    if ([card.title isEqualToString:@"Tap to open stack"]) {
```

Comment:
AppDelegate should have only UIApplicationDelegate methods.
maybe we should move cardStackControllerWillOpen somewhere else ?